projects/sandbox/plots/plots/1D_param_hist.py

```diff
@@ -0,0 +1,119 @@
+import logging
+from pathlib import Path
+from typing import Optional
+
+import numpy as np
+from bokeh.io import save
+from bokeh.layouts import gridplot
+from bokeh.plotting import figure
+from typeo import scriptify
+
+from aframe.analysis.ledger.events import RecoveredInjectionSet
+from aframe.analysis.ledger.injections import InjectionParameterSet
+from aframe.logging import configure_logging
+
+
+@scriptify
+def main(
+    foreground_file: Path,
+    rejected_params: Path,
+    output_fname: Path,
+    log_file: Optional[Path] = None,
+    verbose: bool = False,
+):
+    configure_logging(log_file, verbose)
+
+    logging.info("Reading in inference outputs")
+
+    foreground = RecoveredInjectionSet.read(foreground_file)
+    rejected = InjectionParameterSet.read(rejected_params)
+    all_params = {
+        k: np.concatenate(((foreground[k], rejected[k])))
+        for k in rejected.keys()
+    }
+
+    labels = [
+        "mass_1",
+        "mass_2",
+        "redshift",
+        "psi",
+        "a_1",
+        "a_2",
+        "tilt_1",
+        "tilt_2",
+        "phi_12",
+        "phi_jl",
+        "ra",
+        "dec",
+        "theta_jn",
+        "snr",
+    ]
+
+    logging.info("Read in:")
+    logging.info(f"\t{len(foreground)} foreground events")
+    logging.info(f"\t{len(rejected)} rejected events")
+    logging.info(f"\t{len(all_params)} total events")
+
+    hist_dict_fore = {
+        k: np.histogram(foreground[k], bins=100, density=True) for k in labels
+    }
+    hist_dict_rej = {
+        k: np.histogram(rejected[k], bins=100, density=True) for k in labels
+    }
+    hist_dict_all = {
+        k: np.histogram(all_params[k], bins=100, density=True) for k in labels
+    }
+
+    fore_plots = []
+    rej_plots = []
+    all_plots = []
+    for lab in labels:
+        fore = create_figure(lab, data=hist_dict_fore, dist="Fore")
+        rej = create_figure(lab, data=hist_dict_rej, dist="Rej")
+        all = create_figure(lab, data=hist_dict_all, dist="All")
+
+        fore_plots.append(fore)
+        rej_plots.append(rej)
+        all_plots.append(all)
+
+    grid = gridplot(
+        [fore_plots, rej_plots, all_plots], toolbar_location="right"
+    )
+    save(grid, filename=output_fname)
+
+
+def create_figure(x_var: str, data: dict, dist: str):
+    if dist == "Fore":
+        title = "Normalized Foreground Distribution of {}".format(x_var)
+    elif dist == "All":
+        title = "Normalized Foreground + Rejected Distribution of {}".format(
+            x_var
+        )
+    elif dist == "Rej":
+        title = "Normalized Rejected Distribution of {}".format(x_var)
+    else:
+        logging.info("Not a Supported Distribution")
```

Comment:
Ah, sorry, this is my bad. I saw the `print` statement and didn't pay attention to what it was doing. Here, you should raise an error, so something like: `raise ValueError(f"{dist} is not a supported distribution")`. That will interrupt the script and print out the message